src/utils/Config.tsx

```diff
@@ -13,17 +14,18 @@ export const baseStrokeThickness = 1;
 export const baseTranslateOffset = 12;
 export const baseZoomThreshold = 16;
 export const defaultMarkerColor = "#3786C2";
-export const evenRowColor = "#ededed";
```

Comment:
Related to "Refactored table rows coloring to have dark odd and light even rows"
Removed unused constants since CSS was now using those values